server/models/block.go

```diff
@@ -0,0 +1,38 @@
+package models
```

Comment:
We should make these `models` types easily available for import into client projects. What do you think about moving them to the top level of the repo? That way they can import `github.com/gochain-io/explorer` and use `explorer.Block`, rather than `github.com/gochain-io/explorer/server/models` with `models.Block` (or an import alias).